launch/launch/launch_description.py

```diff
@@ -76,6 +76,25 @@ def get_launch_arguments(self, conditional_inclusion=False) -> List[DeclareLaunc
         """
         Return a list of :py:class:`launch.actions.DeclareLaunchArgument` actions.
 
+        See :py:method:`get_launch_arguments_with_launch_descriptions()`
+        for more details.
+        """
+        return [
+            item[0] for item in
+            self.get_launch_arguments_with_include_launch_descriptions(conditional_inclusion)
+        ]
+
+    def get_launch_arguments_with_include_launch_descriptions(
+        self, conditional_inclusion=False
+    ) -> List[DeclareLaunchArgument]:
+        """
+        Return a list of (:py:class:`launch.actions.DeclareLaunchArgument`, Optional[List[:py:class:`launch.actions.IncludeLaunchDescription`]]) pairs.
```

Comment:
nitpick: should probably wrap this line somehow.

I see you're suppressing that warning, but I don't know why this cannot be wrapped.

Reply:
If I wrap the line I get:

>      82 ./launch/launch_description.py:95:1: D205 1 blank line required between summary line and description
>      86 ./launch/launch_description.py:95:1: D400 First line should end with a period

I solved it by shortening the summary line, see https://github.com/ros2/launch/pull/556/commits/2c49f810cd6bc7b851ab785bb9ade9146aabcde3.